### ISSUE TITLE
Undefined control sequence: "\dia"

### ISSUE THREAD

**Issue by JonasOlson** [KingArhturII/macbeth#1]
Is the command "\dia" from some dialogue package or perhaps some command you have defined yourself? In any case, its definition seems to have gone missing.

```
! Undefined control sequence.
l.62            \dia
          {\oldstylenums{1}.} When shall we three meet againe? \\ In Thunder...
```

**Comment by KingArhturII** [comment]
I believe it comes from the `sides` document class, but I'll look into it when I get home.

**Comment by JonasOlson** [comment]
Fixed in commit 9347508f8377b17186ad43b8772a19552c96350f.
